jwt.d.ts

```diff
@@ -117,7 +117,7 @@ export interface JWT {
 
 export type { JwtHeader } from 'fast-jwt'
 
-export const fastifyJWT: fastify.FastifyPluginCallback<FastifyJWTOptions>
+declare const fastifyJWT: fastify.FastifyPluginCallback<FastifyJWTOptions>

```

Comment:
```suggestion
declare const fastifyJwt: fastify.FastifyPluginCallback<FastifyJWTOptions>
```

Reply:
thanks, you were right. The problem was in the naming of exported constant